actionlist/app/components/display_list.js

```diff
@@ -28,8 +29,9 @@ const mapStateToProps = state => ({
   dataSource: ds.cloneWithRows(state.actionItemsReducer.dataSource),
 });
 
-export default connect(mapStateToProps)(DisplayList);
-
 DisplayList.propTypes = {
   dataSource: React.PropTypes.shape({}).isRequired,
 };
+
+export default connect(mapStateToProps)(DisplayList);
```

Comment:
@PrashanthKumarM Are you talking about whether it's better to have the default export at the bottom of the file? If so, I think it's pretty standard for it to go there.